pages/doc/wavefront_spring_boot3.md

```diff
@@ -0,0 +1,450 @@
+
+* The easiest way to get started is to create a new project on [start.spring.io](https://start.springboot.io).
+  - Select Spring Boot 2.3 or later and your favorite build system
+  - Include at least the `Wavefront` entry.
+  - If you want to opt-in for tracing support, add the `Spring Cloud Sleuth` entry as well.
+* If you already have a Spring Boot application, you can also use [start.spring.io](https://start.springboot.io) to explore the project from your browser. You can determine the setup for the Spring Boot generation that your  project is using.
+
+Here are some things to know before you start:
+
+* **Ingestion Method**: Wavefront for Spring Boot sends data to Wavefront via [direct ingestion](direct_ingestion.html) by default. You can [configure your application to send data via the Wavefront proxy](#proxy).
+* **Target**: Wavefront for Spring Boot sends data to the Wavefront Freemium instance by default. You can [configure your application to send data to your Wavefront instance](#wf_instance).
+* **Account**: By default the starter sends you to the Freemium instance,  auto-negotiates an account, and saves the API token in the `~/.wavefront_freemium` file in your home directory. If you customize the starter to go to your Wavefront instance (see Step 2 below) you need to include an API token for that instance. -->
+
+<!---
+![Ways to get started with the Wavefront for Spring Boot offering](/images/spring_boot_getting_started.png)--->
+
+## Getting Started
+
+Getting started is easy. Here are some things to know before you start:
+
+* **Ingestion Method**: Wavefront for Spring Boot sends data to Wavefront via [direct ingestion](direct_ingestion.html) by default. You can [configure your application to send data via the Wavefront proxy](#proxy).
+* **Target**: Wavefront for Spring Boot sends data to the Wavefront Freemium instance. You can change the default behavior and  [configure your application to send data to your Wavefront instance](#step-2-optional-specify-your-wavefront-instance).
+* **Account**: By default, the starter sends you to the Freemium instance, auto-negotiates an account, and saves the API token in the `~/.wavefront_freemium` file in your home directory. If you customize the starter to go to your Wavefront instance (see Step 2 below), you need to include an API token for that instance.
+
+### Prerequisites for Wavefront Spring Boot Starter
+
+* Spring Boot 3.0.0 or above
+  <a name="versionCompatibility"></a>
+  <table style="width: 70%;">
+  <tbody>
+  <thead>
+  <tr><th width="35%">Spring Boot Version</th><th width="35%">Wavefront for Spring Boot Version</th></tr>
+  </thead>
+  <tr>
+  <td>3.0.x</td>
+  <td>3.0.x</td>
+  </tr>
+  </tbody>
+  </table>
+* Java 17 or above
+* Maven 3.5+ or Gradle 7.5 or later
+  <br/>See [System Requirements](https://docs.spring.io/spring-boot/docs/3.0.x/reference/html/getting-started.html#getting-started.system-requirements) in the Spring Boot documentation.
```

Comment:
<br/>See the [System Requirements](https://docs.spring.io/spring-boot/docs/3.0.x/reference/html/getting-started.html#getting-started.system-requirements) in the Spring Boot documentation.